content/en/Integrations/Azure DevOps/integrate-with-azure-devops.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Integrate with Azure DevOps"
+linkTitle: "Integrate with Azure DevOps"
+weight: 50
+description: >
+  Connect your Azure DevOps environment to your Cobalt Organization in order to use the integration to sync pentest findings as work items.
+---
+
+## Set Up Connection
+
+As an [Organization Owner](/platform-deep-dive/collaboration/user-roles/#organization-owner) or [Member](/platform-deep-dive/collaboration/user-roles/#organization-member), you can set up a connection to your Azure DevOps environment by: 
+
+1. In Cobalt, go to **Integrations** > **Azure DevOps Boards**.
+2. On the **Connection** tab, click **Add Connection**.
+3. In the **Configure Connection with Azure DevOps Boards** form of the overlay, enter:
+   - OAuth 2.0 grant type
+   - Organization: Your Azure DevOps Organization ID (`https://dev.azure.com/{yourOrganization}`)
+   - API Version: 7.0
+   - Username: Your account email address in Azure DevOps
+   - Personal Access Token: Your Azure DevOps [personal access token](https://learn.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate?view=azure-devops&tabs=Windows)
+4. Click **Done**.
```

Comment:
They need to click the "connect" button after entering their info before they click the done button.